src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/workspace_cloner.cljs

```diff
@@ -59,11 +61,12 @@
      (if-let [fails (input/validate refs "name")]
        (swap! state assoc :validation-error fails)
        (let [name (input/get-text refs "name")
-             project (nth (:billing-projects props) (int (:selected-project @state)))]
+             project (nth (:billing-projects props) (int (:selected-project @state)))
+             protected? (react/call :checked? (@refs "protected-check"))]
          (swap! state assoc :working? true :validation-error nil :error nil)
          (endpoints/call-ajax-orch
            {:endpoint (endpoints/clone-workspace (:workspace-id props))
-            :payload {:namespace project :name name}
+            :payload {:namespace project :name name :attributes {} :isProtected protected?}
```

Comment:
I just tried it - modified the DOM via dev tools, submitted a request to clone a protected workspace, but unchecked the checkbox. I did not get an error, but the cloned workspace is still protected.

If that's the case, and is expected behavior, I suggest changing the UI so that if the current workspace is protected, we don't even show a checkbox; we just show the "Cloned workspace will automatically be protected because this workspace is protected" message. I don't feel too strongly about the UI design.

Please confirm expected behavior of the back end, though? (Throw an error vs. always do the right thing)